About.html

```diff
@@ -0,0 +1,125 @@
+      <i class="fa-solid fa-bars" id="menuOpen"></i>
+      <ul id="menuList">
+        <a href="./Main.html#topnav" class="mlogo1">LAGF</a>
+        <i class="fa-solid fa-times" id="menuClose"></i>
+        <div class="droplist">
+          <li><a href="#about">About</a></li>
+          <li><a href="./Main.html#mainProgram" id=>Program</a></li>
+          <li><a href="./Main.html#bookBtn" id=>Join</a></li>
+          <li><a href="./Main.html#sponsorTitle">Sponsor</a></li>
+          <li><a href="./Main.html">Home</a></li>
+          <li><a href="./Main.html#guest">Speakers</a></li>
+          <li><button type="button" class="campaignBtn">LAGF Campaign</button></li>
+        </div>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section id="about">
+      <div id="aboutwrap">
+        <h3 id="hello" class="deskhello">"Hello! Sharing world"</h3>
+        <h1 id="aboutFest">LUXURY AUTO GLOBAL FESTIVAL 2023</h1>
+      </div>
+      <div id="aboutDetail">
+        <p>The Auto Fest brings together the community of experts, manufacturers, owners, and riders who comprise the
+          Luxury Auto Summit affiliate network in a different country every two.</p>
+        <p class="deskp">This year we hope to expand the list if invitations to companies and individuals who want to
+          work with us on
+          shared projects that advance the cause of the Auto Fest.</p>
+      </div>
+      <div>
+        <p id="contact">Please contact us per Email for any further questions about Luxury Auto Fest 2023!</p>
+        <a href="mailto:tolugeorges@gmail.com" id="contactInfo">Autofest2023@lucas.org</a>
+      </div>
+    </section>
+
+    <section id="logoDiv">
+      <div>
+        <h3 class="titles">Luxury Auto Global Festival 2023 Logo</h3>
+        <div class="bar2"></div>
+      </div>
+      <div>
+        <p id="logoDesc">Our logo is a combination of the Luxury Auto Global Festival 2023 and the Luxury Auto Global
+          Festival 2023.</p>
+      </div>
+      <div id="logoBox">
+        <img src="Assets/bd9676507b5e4142a89209019ec1d0c8__1_-removebg-preview.png" alt="Luxury Auto Logo">
+      </div>
+      <i class="hrGrey"></i>
+    </section>
+
+    <section id="history">
+      <div>
+        <h4 class="titles">See the past Luxury Auto Fest</h4>
+        <div class="bar2"></div>
+        <p id="historyDesc">Take a look at the last two Luxury Auto Festivals which took place in Lagos Nigeria and
+          Abuja.</p>
+      </div>
+      <div id="oldConContainer">
+        <div class="oldFest" id="f21">
+          <h4 class="previousYear">2021</h4>
+          <p class="previousDetail">Luxury Auto Fest 2021 in Lagos</p>
+        </div>
+        <div class="oldFest" id="f22">
+          <h4 class="previousYear">2022</h4>
+          <p class="previousDetail">Luxury Auto Fest 2022 in Abuja</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="sponsor">
+      <div>
+        <h4 id="sponsorTitle">Partner</h4>
```

Comment:
- [optional] please the desktop about page does not have a partner section. Kindly fix that.